style.css

```diff
@@ -385,10 +416,12 @@ div.data-dashbord {
   width: 100%;
  */
 
-  margin-left: auto;
-  margin-right: auto;
+  /* margin-left: auto; */
+
+  /* margin-right: auto; */
+
```

Comment:
- [ ] I kindly suggest to avoid leaving commented code in your code base. This is because Leaving commented-out code in source code is bad practice, as it takes up space, causes confusion, and leads to maintenance issues if it's not removed 😞.